### PR TITLE
ALC no longer working (fix: #11001)

### PR DESCRIPTION
**Pull request by baiti**
fixes broken ALC functionality

### CODE REVIEW

File: main/templates/keys.xml [hunk 1]
```diff
@@ -30,4 +30,7 @@
     <!-- Geocaching.su -->
     <string name="su_consumer_key" translatable="false">@su.consumer.key@</string>
     <string name="su_consumer_secret" translatable="false">@su.consumer.secret@</string>
+
+    <!-- Geocaching.com public ALC API -->
+    <string name="alc_consumer_key" translatable="false">@alc.consumer.key@</string>
```
Comment (bekuno) [inline main/templates/keys.xml:35]: As documented in https://github.com/cgeo/cgeo#api-keys-installation the key management has two parts - in the `main/template/keys.xml` you define the part for using the key in the code. And in `templates/private.properties` you define the point where the user should put his keys. 

The workflow described in the Readme works.

@baiti Please add the missing entry in `templates/private.properties` and maybe also a short text in the Readme.

Comment (baiti) [reply main/templates/keys.xml:35]: > As documented in https://github.com/cgeo/cgeo#api-keys-installation the key management has two parts - in the `main/template/keys.xml` you define the part for using the key in the code. And in `templates/private.properties` you define the point where the user should put his keys.
> 
> The workflow described in the Readme works.

Yes, it works, but it confused me. `private.properties` we take from `./templates/` and `keys.xml` we update in `./main/templates/`. 
> 
> @baiti Please add the missing entry in `templates/private.properties` and maybe also a short text in the Readme.

first part is done and commited. Not sure about what to write into the READ.me.

**Comment by moving-bits** [comment]
Setting the "do not merge" label for now, as we have to inject the key in our CI first

**Comment by Lineflyer** [comment]
Preconditions for this PR are now met and a local test on my machine was succesful.
Merging now.

The CI still needs to be prepared for our productive/beta/nightly releases. I informed @mucek4 accordingly.

**Pull request merged**